catalog.html

```diff
@@ -0,0 +1,1394 @@
+                  <li class="chip-list__chip-item">
+                    <button
+                      type="submit"
+                      class="chip-list__chip-link button button__secondary button__small"
+                    >
+                      UI kit
+                      <svg class="chip-list__chip-image icon__small">
+                        <use xlink:href="./images/icons/sprite.svg#close" />
+                      </svg>
+                    </button>
+                  </li>
+                  <li class="chip-list__chip-item">
+                    <button
+                      type="submit"
+                      class="chip-list__chip-link button button__secondary button__small"
+                    >
+                      UI kit
+                      <svg class="chip-list__chip-image icon__small">
+                        <use xlink:href="./images/icons/sprite.svg#close" />
+                      </svg>
+                    </button>
+                  </li>
+                </ul>
+              </div>
+              <button type="submit" class="hidden">Apply</button>
+            </form>
+          </header>
+          <div class="catalog__viewer">
+            <ul class="viewer__catalog-cards list">
+              <li class="catalog-cards__list-item">
+                <a href="#" class="catalog-cards__link" target="_blank">
+                  <article
+                    class="catalog-cards__catolog-card container__card behavior__card"
+                  >
+                    <div class="catolog-card__image-container">
+                      <img
+                        src="./images/popular-products/lens-chronicles.webp"
+                        alt="lens chronicles asset"
+                        class="catolog-card__image"
+                        width="325"
+                        height="260"
+                      />
+                      <span
+                        class="catolog-card__label label label__sold-out heading__section"
+                      >
+                        Sold out
+                      </span>
+                    </div>
+                    <div class="catolog-card__information">
+                      <h3 class="catolog-card__title heading__card-title">
+                        Lens Chronicles
+                      </h3>
+                      <p class="catolog-card__description">
+                        Photography · Creative
+                      </p>
+                    </div>
+                    <div class="catolog-card__cart-manager list">
+                      <strong class="catolog-card__price heading__card-title"
+                        >€00.00</strong
+                      >
+                    </div>
+                  </article>
+                </a>
+                <form
+                  action="#"
+                  method="post"
+                  class="catolog-card__buttons list"
+                >
+                  <button
```

Comment:
Here the same - we need to specify title or label or add hidden text, to make buttons accessible.

Reply:
Fixed 🥇